gerrydb/repos/column.py

```diff
@@ -120,15 +120,18 @@ def update(
     @online
     def set_values(
         self,
-        path_or_col: Union[Column, str],
+        path: Optional[str],
         namespace: Optional[str] = None,
         *,
+        col: Optional[Column] = None,
         values: dict[Union[str, Geography], Any],
     ) -> None:
         """Sets the values of a column on a collection of geographies.
 
         Args:
-            path_or_col: Short identifier for the column or a `Column` metadata object.
+            path: Short identifier for the column. Only this or `col` should be provided.
```

Comment:
Can you update this comment about the behavior of the function when both path and col are passed?